package.json

```diff
@@ -4,39 +4,35 @@
     "url": "https://github.com/linterjs/core/issues"
   },
   "dependencies": {
-    "common-tags": "^1.8.0",
```

Comment:
Using ~ will only allow PATCH version updates, while ^ will allow MINOR version updates. I guess I'm ok with only allowing PATCH updates, however if this package will be using `@linter/cli` in the future that will have to be using ^ as our packages will never have changes to PATCH as that will always be 0.

Reply:
Oh sorry! you are right, but only for `dependencies`.
But we should use ~ for `devDependencies`

I'm always using ~ but I'm using them in frontend projects, and these are not dependencies for other projects ^^